scripts/docker/Dockerfile.gcc.full

```diff
@@ -32,15 +32,15 @@ RUN apt-get update -y && \
         python3-setuptools \
         python3-wheel && \
     rm -rf /var/lib/apt/lists/*
-RUN pip3 install virtualenv
-# CMake version 3.13.4
+RUN pip3 install virtualenv pre-commit
+# CMake version 3.15.1
```

Comment:
Is 3.15.1 the new minimum for cmake or only in docker?

Reply:
no, my fault. I now set this to 3.12.4.